src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationOnSameLineCheck.java

```diff
@@ -56,19 +56,75 @@
  * &lt;module name=&quot;AnnotationOnSameLine&quot;/&gt;
  * </pre>
  * <p>
- * Example to allow annotations on the same line
+ * Example:
  * </p>
  * <pre>
- * &#64;Override public int toString() { ... } // no violations
- * &#64;Before &#64;Override public void set() { ... } // no violation
+ * class Foo {
+ *
+ *   &#64;SuppressWarnings("deprecation")  // violation, annotation should be on the same line
+ *   public Foo() {
+ *   }
+ *
+ *   public void fun1() {
+ *   }
```

Comment:
Please remove this code, it shows nothing

Reply:
This method ```fun1()``` is Overridden in the child class. Should I still remove it?